android/app/src/main/java/org/fossasia/openevent/utils/ShowNotificationSnackBar.java

```diff
@@ -38,18 +38,31 @@ public ShowNotificationSnackBar(Context context, View view, SwipeRefreshLayout s
     public abstract void refreshClicked();
 
     public Snackbar showSnackBar(){
-        snackbar = Snackbar.make(view, R.string.waiting_for_network, Snackbar.LENGTH_INDEFINITE).setAction(R.string.snackbar_refresh_action,
-                new View.OnClickListener() {
-                    @Override
-                    public void onClick(View v) {
-                        if (swipeRefreshLayout!=null)
-                        {
-                            swipeRefreshLayout.setRefreshing(true);
+        if(view!=null) {
+            snackbar = Snackbar.make(view, R.string.waiting_for_network, Snackbar.LENGTH_INDEFINITE).setAction(R.string.snackbar_refresh_action,
+                    new View.OnClickListener() {
+                        @Override
+                        public void onClick(View v) {
+                            if (swipeRefreshLayout != null) {
+                                swipeRefreshLayout.setRefreshing(true);
+                            }
+                            snackbar.dismiss();
+                            refreshClicked();
                         }
-                        snackbar.dismiss();
-                        refreshClicked();
-                    }
-                });
+                    });
+        }else{
```

Comment:
Don't duplicate code

Create a local `View` variable and assign it any non null view instance and pass that as an argument to the Snackbar method